tools/breaking-changes/api-extractor.json

```diff
@@ -0,0 +1,341 @@
+/**
+ * Config file for API Extractor.  For more info, please visit: https://api-extractor.com
+ */
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
```

Comment:
I think we can remove those comments in this file. First, we cannot add such comments in json; and that schema link really give all info about the file.

Reply:
IT's meant do document the different configs we could apply to this file.   I thought it could be useful down the road.

Reply:
Comments are not permitted in json; and we can get all info from that schema link, right?

Reply:
The Microsoft api extractor tool generates the config file like this (with all the comments), so I presumed that having comments in there was fine.